### ISSUE TITLE
SVG PARSING ERROR:"Failed to find gradient fill: Pattern" (for patterns)

### ISSUE THREAD

**Issue by Joxit** [mapnik/mapnik#3784]
Hello,
I'm trying to use patterns in SVG but mapnik sends me errors.

Here is the mozilla example for [SVG patterns](https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorial/Patterns) I used : 
```svg
<svg width="200" height="200" xmlns="http://www.w3.org/2000/svg">
  <defs>
    <linearGradient id="Gradient1">
      <stop offset="5%" stop-color="white"/>
      <stop offset="95%" stop-color="blue"/>
    </linearGradient>
    <linearGradient id="Gradient2" x1="0" x2="0" y1="0" y2="1">
      <stop offset="5%" stop-color="red"/>
      <stop offset="95%" stop-color="orange"/>
    </linearGradient>

    <pattern id="Pattern" x="0" y="0" width=".25" height=".25">
      <rect x="0" y="0" width="50" height="50" fill="skyblue"/>
      <rect x="0" y="0" width="25" height="25" fill="url(#Gradient2)"/>
      <circle cx="25" cy="25" r="20" fill="url(#Gradient1)" fill-opacity="0.5"/>
    </pattern>
  </defs>

  <rect fill="url(#Pattern)" stroke="black" width="200" height="200"/>
</svg>
``` 

And the full error is : 

```
Mapnik LOG> 2017-10-16 08:58:17: SVG PARSING ERROR:"Failed to find gradient fill: Pattern"
```

- **OS**: Debian Stretch 9.2
- **mapnik**: v3.0.15
- **node-mapnik**: 3.6.2

**Comment by artemp** [comment]
@Joxit - SVG `pattern` element is not supported, see : https://github.com/mapnik/mapnik/wiki/SVG-support
If you feel it should be supported, please, create a separate issue and I'll try to find time to implement it. Thanks!

**Comment by artemp** [comment]
Actually, I think we need to fix error message at least, re-opening to track it here.

**Comment by artemp** [comment]
Error message fixed in 831e353c5dd49f153a9e10c58415ac650fe88c35
